src/XMakeTasks/UnitTests/ResolveAssemblyReference_Tests.cs

```diff
@@ -8248,7 +8248,7 @@ public void SimulateCreateProjectAgainstWhidbey(string fxfolder)
         [Fact]
         public void SimulateCreateProjectAgainstWhidbey()
         {
-            SimulateCreateProjectAgainstWhidbey(ToolLocationHelper.GetPathToDotNetFramework(TargetDotNetFrameworkVersion.Version45));
+            SimulateCreateProjectAgainstWhidbeyInternal(ToolLocationHelper.GetPathToDotNetFramework(TargetDotNetFrameworkVersion.Version45));
```

Comment:
This was failing for me with `System.InvalidOperationException : The test method expected 1 parameter value, but 0 parameter values were provided`. No clue how this wasn't caught before. Maybe upgraded xunit? Seems like a test fixture issue, so I renamed the overload and it worked.